lua/cmp_path/init.lua

```diff
@@ -27,12 +27,8 @@ source.complete = function(self, params, callback)
     return callback()
   end
 
-  local stat = self:_stat(dirname)
-  if not stat then
-    return callback()
-  end
```

Comment:
I think we still need this check. Why do you remove this?

Reply:
No, it is redundant `fs_scandir` will fail if a directory doesn't exist or the path points to a file, and the rest of the `_candidates` method won't be executed.